storage/ceph.py

```diff
@@ -1,14 +1,22 @@
 import pyarrow.parquet as pq
-import s3fs
+import s3fs, datetime, os, s3, local
 
 (ceph_key, ceph_secret, ceph_endpoint) = (os.getenv('CEPH_KEY'), os.getenv('CEPH_SECRET'), os.getenv('CEPH_ENDPOINT'))
 client_kwargs = {'endpoint_url': ceph_endpoint}
 
 
 def get_dataset(key):
-    s3 = s3fs.S3FileSystem(secret=ceph_secret, key=ceph_key, client_kwargs=client_kwargs)
-    return pq.ParquetDataset(key, filesystem=s3).read_pandas().to_pandas()
+    filesystem = s3fs.S3FileSystem(secret=ceph_secret, key=ceph_key, client_kwargs=client_kwargs)
```

Comment:
Let the ceph module keep operating on ceph. [__init__](https://github.com/RedHatInsights/aicoe-insights-clustering/blob/master/storage/__init__.py) will choose which storage to operate on. The idea is that every storage module uses the same API and acts transparently.

Reply:
Sorry could you clarify. Should I revert lines 9 and 10? I changed the variable s3 to filesystem because I imported the s3 module and the names conflict.

Reply:
Have a look at the __init__ link I've provided. You'll wont have s3 and ceph imported at the same time. 
```python
import storage
```
this will either use s3, ceph or local. In that order.

Reply:
Oh, my bad. those 2 changes seem to be fine.